command/import_test.go

```diff
@@ -274,691 +274,6 @@ func TestImport_providerConfigWithVarFile(t *testing.T) {
 	testStateOutput(t, statePath, testImportStr)
 }
 
-/*
```

Comment:
While I'm also not a fan having commented out code sitting here, especially since it can quickly grow stale, are these useful for documenting what _should_ work? I'm having trouble seeing why these were added in the first place.